app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+<div class="top_login">
+<button><%= link_to "Back to Log in", new_session_path(resource_name),
+             class: "sign_up_btn" %></button>
+<h2>Sign up</h2>
+<%= f.submit "Sign up" %>
+</div>
+
+  
+<div class="body_login">
+  <div class="field">
+    <%= f.text_field :name, autofocus: true, placeholder: "Name", class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autocomplete: "email", placeholder: "Email",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
```

Comment:
- [ ] Good job so far 👏 👏 But as requested by the previous reviewer, you need to improve your design in order to match the given design. To be specific, I have highlighted the key things that I suggest that you work on here on the registration page. 👇 
**Your Design**
![image](https://user-images.githubusercontent.com/90222110/209002208-b4321ca7-6c90-42cb-a565-7897fc5ab903.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/209002390-f3f643ed-91f3-4927-984e-2b3d36e492ce.png)

Please add the back arrow icon for the back to previous page button instead of text 👍 
Please add the correct style to the button on the top right corner ('Next') but I think you can call it 'register' instead 😉 
Please remove the shown form input labels as they are not in the design given 🏃